_overview/overview-of-spinnaker.md

```diff
@@ -4,39 +4,63 @@ title: Overview of Armory Spinnaker
 
 Spinnaker is an open source multi-cloud Continuous Delivery and Deployment tool started by Netflix for fast and stable deployments.
 
 Spinnaker’s flexible pipeline model allows for easy customization and enhancement in addition to facilitating smart pipeline management to serve a variety of cloud deployment needs.
 
-As of today, Spinnaker can deploy to and manage clusters simultaneously across both Amazon Web Services (AWS), Kubernetes and Google Cloud Platform (GCP) with full feature compatibility for those cloud providers. Spinnaker can also deploy to Cloud Foundry, and support for full integration with Microsoft Azure is currently underway.
+Spinnaker not only enables businesses to move to the cloud but makes it easier for them to adopt the cloud’s advantages.
+
+Spinnaker encourages:
+
+**Safety and Speed**
+
+Today’s world revolves around software and services working reliably and continuously -- the internet is accessible 24 hours a day, and users expect 100% uptime. The cost of business services experiencing downtime, planned or unplanned, is only growing. This means that companies need to be able to deploy software in a way that is safe. In the past, releases were big large monoliths, and ensuring uptime (or deployment safety) meant a long time between each release, including maybe even extended code freezes. A company that wants to maintain a stable environment can become averse to pushing out new features, leading to a slowdown in innovation. This tradeoff is getting more and more difficult to justify.
 
-![Spinnaker's Responsibilities](/assets/images/armory_what_is_spinnaker.png)
+Spinnaker enables safer and faster deployments by facilitating the following:
 
+- **Immutable infrastructure** that builds trust by making sure infrastructure matches an understood and explicit pattern that does not change once it is deployed. If changes are required, a brand new instance gets deployed. Having unique instances for each build enables the use of different deployment strategies, which are another benefit of Spinnaker.
+- **Deployment strategies** to fit your needs and infrastructure. The strategies include the following:
+  - **Blue/Green (Red/Black)**:  An easy way to think about this is that it is similar to active-active high availability. You have two instances of your deployment running concurrently, the production build and a new one. Once you feel confident that the newer build is stable, traffic is shifted all at once from the old deployment to the new one. A configurable number of server groups are maintained, which allows for easy rollback in case of issues.
+  - **Rolling Blue/Green (Red/Black)**:  Similar to Blue/Green, but traffic is gradually shifted from the older deployment to the new one.
```

Comment:
this is experimental and doesn't work much at all

Reply:
also may want to call out what the "difference" is between blue/green and red/black

Reply:
I'm just going to drop red/black. Our field folks chimed in and most people know it as blue/green.